katpoint/projection.py

```diff
@@ -768,9 +928,11 @@ def plane_to_sphere_ssn(az0, el0, x, y):
     num = sin_el0 * cos_theta - cos_el0_cos_daz * y
     den = sin_el0 * y + cos_theta * cos_el0_cos_daz
     # Ensure that cos(el) denominator term is positive to have abs(el) <= 90 degrees
-    if np.any(den < -1e-12):
-        raise ValueError('The y coordinate causes el to be outside range of +- pi/2 radians')
+    check = 'The y coordinate causes el to be outside range of +- pi/2 radians'
+    den = OutOfRange.treat(den, check, lower=0.0)
     el = np.arctan2(num, den)
+    # Ensure that az is NaN when el is NaN
+    az *= np.where(np.isnan(el), np.nan, 1.0)
```

Comment:
Wouldn't `az = np.where(np.isnan(el), np.nan, az)` remove the need for multiplication?

Reply:
Thanks, forgot about that! Fixed.